Arrangement_on_surface_2/doc/Arrangement_on_surface_2/fig_src/arr_segs.tex

```diff
@@ -2,98 +2,95 @@
 \input{header}
 \pagestyle{empty}
 \begin{document}
-\psscalebox{1.8}{\arrset{0.8}
-  \pspicture[](-0.2,-0.2)(12.2,12.2)
-  %\psgrid(0,0)(12,12)
-  \arrMainVertex( 1, 3){a}
-  \arrMainVertex( 2, 4){b}
-  \arrMainVertex( 0, 6){c}
-  \arrv[90](5,11){1}
-  \arrMainVertex(9,11){f}
-  \arrMainVertex(12, 8){g}
-  \arrv[-60](8,8){2}
-  \arrMainVertex( 8, 4){i}
-  \arrMainVertex( 6, 4){j}
-  \arrMainVertex( 6, 2){k}
-  \arrMainVertex( 4, 0){m}
-  \arrMainVertex( 9, 0){n}
-  \arrMainVertex(11, 2){o}
-  \arrMainVertex(12, 4){p}
-  \arrMainVertex(12, 0){q}
-  \arrMainVertex( 4, 2){r}
-  \arrMainVertex( 3, 3){s}
-  \arrMainVertex( 5, 3){t}
+\begin{tikzpicture}[scale=1.0,thick]
+  \node[point] at ( 1, 3) (a) {};
+  \node[point] at ( 2, 4) (b) {};
+  \node[point] at ( 0, 6) (c) {};
+  \node [point,label={[label distance=0pt]90:{$v_1$}}] at ( 5,11) (1) {};
+  \node[point] at ( 9,11) (f) {};
+  \node[point] at (12, 8) (g) {};
+  \node [point,label={[label distance=0pt]-45:{$v_2$}}] at ( 8, 8) (2) {};
+  \node[point] at ( 8, 4) (i) {};
+  \node[point] at ( 6, 4) (j) {};
+  \node[point] at ( 6, 2) (k) {};
+  \node[point] at ( 4, 0) (m) {};
+  \node[point] at ( 9, 0) (n) {};
+  \node[point] at (11, 2) (o) {};
+  \node[point] at (12, 4) (p) {};
+  \node[point] at (12, 0) (q) {};
+  \node[point] at ( 4, 2) (r) {};
+  \node[point] at ( 3, 3) (s) {};
+  \node[point] at ( 5, 3) (t) {};
```

Comment:
Don't you need to update as well the `png` versions of those pictures, so that they are updated in the CGAL documentation?